### ISSUE TITLE
Change user dashboard

### ISSUE THREAD

**Issue by pinarpink** [elixir-luxembourg/daisy#150]
**Is your feature request related to a problem? Please describe.**
Currently the user dashboard includes boxes that list things like:

- Latest datasets
- Latest projects 

Based on our DAISY use experience with LCSB users, we can say current items in dashboard do not make sense to the regular user.

**Describe the solution requested**

We propose that the dash board includes, instead the following lists:

- My Projects, My Datasets
- Shortcut links to create new .... (Project, Dataset)
- My notifications

**Describe possible alternatives**

This issue should be discussed among team before starting with implementation.

**Other info context**

**Comment by vildead** [comment]
Great! After discussion with @evescat, there is more to discuss:
 * My notification - this should be just several upcoming notifications, right?
 * My projects - this lists all projects
   a) I own 
   b) I am local custodian
   c) I am listed in project's personnel or local custodian - this might have two boxes (My projects and Projects I Work On/I am part of/...)

 * grey box with "DAta Infromation SYstem"  is kind of boring. What about to add a picture/ logo and a catch phrase/headline - "DAta Information SYstem for accountability under the GDPR"?
 * there might be Search box with placeholder "Search (Projects, Datasets, Data Declarations, ...)" for super quick navigation.

![daisy_web_home1](https://user-images.githubusercontent.com/51602017/79506964-2c1ee400-8037-11ea-9244-853502488555.png)

**Comment by jLebioda** [comment]
I like the ideas very much!

Regarding the search box - @neoflex, is running the query over different models feasible in solr?

**Comment by jLebioda** [comment]
The shortcuts "add new project" and "add new dataset" could be included in the boxes on the left, e.g:
<img width="1620" alt="Screenshot 2020-04-21 at 17 54 12" src="https://user-images.githubusercontent.com/32834948/79886411-1e7faa80-83f9-11ea-9f63-10f9dd361f8d.png">

**Comment by pinarpink** [comment]
@vildead @jLebioda  both scrshots look great!! I vote for add buttons to be located in respective boxes (as in Jacek's scrshot).  I propose to rename " Upcoming events" to either "Notifications" or "Expiry Alerts"

**Comment by pinarpink** [comment]
@jLebioda @vildead  is there already a branch for this? I am wondering if it is doable (without the quick search.) before the 19 May workshop?

**Comment by jLebioda** [comment]
I will push the branch in a few minutes

Sure, I will do my best

**Comment by jLebioda** [comment]
The feature will be developed on 150-new-dashboard branch (https://github.com/elixir-luxembourg/daisy/tree/150-new-dashboard)

**Comment by jLebioda** [comment]
Currently it looks like that:

<img width="1655" alt="Screenshot 2020-05-08 at 13 27 34" src="https://user-images.githubusercontent.com/32834948/81401507-c2729100-912f-11ea-9276-038172a6cc6a.png">

Collecting all the expiration events for the user might be costly - we might want to avoid having heavy code on index page; possible it could be calculated once per day as a celery task or cron job. BTW, would it be a brand new feature? I tried to find the code to find some code related to getting the events but I couldn't find any

**Comment by pinarpink** [comment]
@jLebioda  hey! that was quick.

**Comment by pinarpink** [comment]
@jLebioda I had a closer look, there's quite a stale code wrt notifications, mail send etc, which has not been tested. My vote is to leave out notifications from the dashboard for now.

**Comment by jLebioda** [comment]
OK, so I'll just hide the right-hand side.

In effect, we'd have:

- new banner
- section My datasets consisting of 5 most recent datasets (user is their local custodian) instead of just last 5 datasets
- section My projects consisting of 5 most recent projects (user is their local custodian) + 5 most recent projects that user is inside personnel list

**Comment by pinarpink** [comment]
@jLebioda agreed!

**Comment by jLebioda** [comment]
<img width="1650" alt="Screenshot 2020-05-08 at 16 37 05" src="https://user-images.githubusercontent.com/32834948/81416547-286c1200-914a-11ea-8d48-fed4790e5984.png">

added a small help block to maintain the alignment

**Comment by pinarpink** [comment]
@jLebioda You were ahead of me with the user guide link.  Looks very good!

**Comment by vildead** [comment]
already done
